src/app/templates/template-dom.component.ts

```diff
@@ -42,7 +42,7 @@ import { DataService } from '../data.service';
         </ngx-datatable-column>
         <ngx-datatable-column name="Gender">
           <ng-template let-column="column" let-sort="sortFn" ngx-datatable-header-template>
-            <span tabindex="1" (click)="sort()" (keyup)="sort()">{{ column.name }}</span>
```

Comment:
This is actually needs a dedicated fix in my opinion. Those spans should be buttons.
Can you do this in another PR?

Reply:
Maybe just remove tabindex=1 here actually. Since we anyway have the sort buttons, it is actually bad to have another focusable element here which does the same.